src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import './index.scss';
```

Comment:
I think using SCSS on this will be super helpful. I guess at some point we can make single files for variables and mixins as we go and then import those into the .scss files. I wonder if we should just put those in the src file, since they will be pulled into multiple .scss files?